### ISSUE TITLE
Login Issue on Chrome/Firefox Browsers

### ISSUE THREAD

**Issue by kjijo** [datastorm-open/shinymanager#182]
When I attempt to log in for the first time, I get "Username or password are incorrect" (after providing the correct credentials). I can log in again if I click "Login" or press "Enter" after seeing this message. This problem has only occurred on machines on which I have previously typed in the wrong username or password. On machines where the correct username and password have always been entered, this is not a problem. I've tested this on both Chrome and Firefox browsers.

**Comment by mardam** [comment]
I can confirm this issue. Running the [example](https://github.com/datastorm-open/shinymanager/tree/master/examples/demo), the user has to wait for a short period of time (maybe 1 second) between finishing to type the password and clicking the login button (or click the login button a second time). The issue seems to be independent of the browser because I observed the issue in the RStudio internal browser, Firefox, Chrome, Edge and Safari.

Session info:

```
R version 4.3.2 (2023-10-31)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 22.04.3 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.10.0 
LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.10.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=de_DE.UTF-8        LC_COLLATE=en_US.UTF-8     LC_MONETARY=de_DE.UTF-8    LC_MESSAGES=en_US.UTF-8    LC_PAPER=de_DE.UTF-8      
 [8] LC_NAME=C                  LC_ADDRESS=C               LC_TELEPHONE=C             LC_MEASUREMENT=de_DE.UTF-8 LC_IDENTIFICATION=C       

time zone: Europe/Berlin
tzcode source: system (glibc)

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] shinymanager_1.0.410 shiny_1.8.0         

loaded via a namespace (and not attached):
 [1] sass_0.4.8        utf8_1.2.4        generics_0.1.3    RSQLite_2.3.1     digest_0.6.34     magrittr_2.0.3    grid_4.3.2        fastmap_1.1.1     blob_1.2.4        R.oo_1.25.0      
[11] jsonlite_1.8.8    R.utils_2.12.2    scrypt_0.1.6      DBI_1.1.3         promises_1.2.1    fansi_1.0.6       scales_1.3.0      textshaping_0.3.7 jquerylib_0.1.4   cli_3.6.2        
[21] rlang_1.1.3       R.methodsS3_1.8.2 ellipsis_0.3.2    bit64_4.0.5       munsell_0.5.0     withr_3.0.0       cachem_1.0.8      tools_4.3.2       memoise_2.0.1     dplyr_1.1.3      
[31] colorspace_2.1-0  ggplot2_3.4.4     httpuv_1.6.14     DT_0.30           vctrs_0.6.5       R6_2.5.1          mime_0.12         lifecycle_1.0.4   htmlwidgets_1.6.4 bit_4.0.5        
[41] ragg_1.2.6        fontawesome_0.5.2 pkgconfig_2.0.3   pillar_1.9.0      bslib_0.6.1       later_1.3.2       gtable_0.3.4      glue_1.7.0        Rcpp_1.0.12       systemfonts_1.0.5
[51] tibble_3.2.1      tidyselect_1.2.0  rstudioapi_0.15.0 billboarder_0.4.1 xtable_1.8-4      htmltools_0.5.7   compiler_4.3.2    askpass_1.2.0     openssl_2.1.1   
```

I figured out that downgrading shiny from 1.8.0 to 1.7.3 solves the issue.

**Comment by bthieurmel** [comment]
Hi. I hope FIX on last version 1.0.510. Due to javascript bind enter event.
